pkgs/development/ocaml-modules/mirage-block-ramdisk/default.nix

```diff
@@ -21,5 +21,9 @@ buildDunePackage rec {
     homepage = "https://github.com/mirage/mirage-block-ramdisk";
     license = licenses.isc;
     maintainers = with maintainers; [ ehmry ];
+    # mirage-block-ramdisk uses an incompatable version of cstruct (<6.1.0) and
+    # is only dependen on by other broken packages. No update has been pushed
+    # since 2022-07-06, so might be abandoned?
+    broken = true;
```

Comment:
Maybe we could create an issue on their repo to fix up cstruct compatibility or package an older version into nixpkgs?

Reply:
Looks like I made a mistake on the comment, the package hasn't been updated since 2020-07-06.

It might be worth adding an old version of cstruct but that would prevent other packages that rely on newer versions of cstruct from building (if I understand how dune works within nix).